### PR TITLE
feat(nginx): nginx image

### PR DESCRIPTION
**Pull request by revolunet**
Add a regular nginx image that doesnt serve `/` on 404 errors.

nginx4spa image makes sense only for some apps that only serve a single index.html with client-side routing

### CODE REVIEW

File: nginx/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+```
+
+Notes:
+
+- `PORT` is optional and default to `80`
+
+To override default configuration, make a local copy of `nginx.conf` and add it to docker build:
+
+```dockerfile
+FROM ghcr.io/socialgouv/docker/nginx4:x.y.z
```
Comment (douglasduteil) [inline nginx/README.md:44]:
```suggestion
FROM ghcr.io/socialgouv/docker/nginx:x.y.z
```

**Review by douglasduteil — APPROVED** [review]
K. So nginx4spa is nginx with a custom nginx.conf ?

**Comment by revolunet** [comment]
> K. So nginx4spa is nginx with a custom nginx.conf ?

the nginx.conf diff is very light indeed : https://github.com/SocialGouv/docker/blob/master/nginx4spa/nginx.conf#L42-L45

**Comment by douglasduteil** [comment]
> Yes, the nginx.conf diff is very light indeed : [`master`/nginx4spa/nginx.conf#L42-L45](https://github.com/SocialGouv/docker/blob/master/nginx4spa/nginx.conf?rgh-link-date=2021-08-17T09%3A23%3A19Z#L42-L45)

We might want to add `include /etc/nginx/conf.d/*` in this `nginx` image so `nginx4spa` will just [add the `/etc/nginx/conf.d/spa-index-fallback.config` 4 lines](https://github.com/SocialGouv/docker/blob/master/nginx4spa/nginx.conf?rgh-link-date=2021-08-17T09%3A23%3A19Z#L42-L45)

File: .github/workflows/nginx.main.workflow.dhall [hunk 2]
```diff
@@ -0,0 +1,132 @@
+concurrency:
```
Comment (douglasduteil) [inline .github/workflows/nginx.main.workflow.dhall:1]: dhall files should be per package

**Review by douglasduteil — CHANGES_REQUESTED** [review]

**Review by douglasduteil — APPROVED** [review]

**Pull request merged**